src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,169 @@ const TitleCardList: React.FC<IProps> = ({
+          width="100%"
+          boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+          _hover={{
+            textDecoration: "none",
+            boxShadow: "0 0 1px var(--eth-colors-primary)",
+            bg: "primary100",
+            color: "black",
+          }}
+        >
+          {image && (
+            <Box
+              as={GatsbyImage}
+              marginTop={1}
+              marginRight={4}
+              minWidth={5}
+              image={image}
+              alt={alt || ""}
+            />
+          )}
+          <Flex flex="1 1 75%" flexDirection="column" marginRight={8}>
+            <Box>{title}</Box>
 
-            <ItemDesc>{description}</ItemDesc>
-          </LeftContainer>
+            <Box fontSize="s" marginBottom="0" opacity="0.7">
+              {description}
+            </Box>
+          </Flex>
           {caption && (
-            <RightContainer>
-              <ItemDesc>{caption}</ItemDesc>
-            </RightContainer>
+            <Flex
+              flex="1 0 25%"
+              alignItems="center"
+              flexWrap="wrap"
+              marginRight={4}
+            >
+              <Box fontSize="s" marginBottom="0" opacity="0.7">
+                {caption}
+              </Box>
+            </Flex>
           )}
-        </ItemLink>
+        </LinkBox>
       ) : (
-        <Item key={idx} onClick={() => clickHandler(idx)}>
-          {image && <Image image={image} alt={alt || ""} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
+        <Flex
+          key={idx}
+          onClick={() => clickHandler(idx)}
+          width="100%"
+          marginBottom="1px"
+          padding={4}
+          textDecoration="none"
+          color="text"
+          justifyContent="space-between"
+          cursor="pointer"
+          boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+          _hover={{
+            boxShadow: "0 0 1px var(--eth-colors-primary)",
+            bg: "primary100",
+            color: "black",
+          }}
+        >
+          {image && (
+            <Box
+              as={GatsbyImage}
+              marginTop={1}
+              marginRight={4}
+              minWidth={5}
+              image={image}
+              alt={alt || ""}
+            />
+          )}
+          <Flex flex="1 1 75%" flexDirection="column" marginRight={8}>
+            <Box>{title}</Box>
 
-            <ItemDesc>{description}</ItemDesc>
-          </LeftContainer>
+            <Box fontSize="s" marginBottom="0" opacity="0.7">
+              {description}
+            </Box>
+          </Flex>
           {caption && (
-            <RightContainer>
-              <ItemDesc>{caption}</ItemDesc>
-            </RightContainer>
+            <Flex
+              flex="1 0 25%"
+              alignItems="center"
+              flexWrap="wrap"
+              marginRight={4}
+            >
+              <Box fontSize="s" marginBottom="0" opacity="0.7">
```

Comment:
Ditto with [line 138](https://github.com/ethereum/ethereum-org-website/pull/8067/files#diff-0f98fcf4d26b7b815c20d7c30103440e8ebba1dd06ef71932ef0fccd7f7bcc60R138)
```suggestion
              <Box fontSize="sm" marginBottom="0" color={descriptionCaptionColor}>
```